amip_interp/amip_interp.F90

```diff
@@ -944,6 +915,12 @@ subroutine amip_interp_init()
         write (unit,nml=amip_interp_nml)
     endif
 
+    if (use_mpp_io) then
```

Comment:
Should the namelist flag get removed too?

Reply:
@thomas-robinson said no

If a user has use_mpp_io in their namelist and we remove it, they will get a useless error message
With this code, they will get a more useful(?) error message ...

Reply:
Yeah I guess a useless nml variable is better than a useless error message